lib/views/widgets/input/input_field.dart

```diff
@@ -1,16 +1,19 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 
 class InputField extends StatelessWidget {
   final String hintText;
-  final bool isNumber;
   final String errorMessage;
+  final TextInputType inputType;
+  final bool isPassword;
```

Comment:
@louisepujante I think no need na for this, we don't have password field to our design.